README.md

```diff
@@ -6,7 +6,7 @@
 
 ----
 
-Kubernetes is an open source system for managing [containerized applications]
+Kubernetes is an open-source system for managing [containerized applications]
```

Comment:
hyphens are used when `open-source` is a compound adjective, rather than a noun (this is the case here).

https://dzone.com/articles/blogging-programmers